2227/2/lib/battles.rb

```diff
@@ -0,0 +1,21 @@
+# Data storage with battles
+class DataStorage
+  def self.find_names_of_the_rappers
```

Comment:
create an instance of the store instead. You can pass the path to the folder as a parameter